src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -231,31 +218,18 @@ void Analyzer::Display() {
         Calculate();
       }
 
-      auto size = ImGui::GetIO().DisplaySize;
-      ImGui::SetNextWindowSize(ImVec2(size.x / 2.5, size.y * 0.9));
-
-      // Show voltage domain diagnostic plots.
-      if (ImGui::BeginPopupModal("Voltage-Domain Diagnostics")) {
-        m_plot.DisplayVoltageDomainPlots();
-        // Button to close popup.
-        if (ImGui::Button("Close")) {
-          ImGui::CloseCurrentPopup();
-        }
-        ImGui::EndPopup();
-      }
-
-      ImGui::SetNextWindowSize(ImVec2(size.x / 2.5, size.y * 0.9));
+      ImGui::SetCursorPosY(endY);
+      ImGui::Separator();
 
-      // Show time domain diagnostic plots.
-      if (ImGui::BeginPopupModal("Time-Domain Diagnostics")) {
-        m_plot.DisplayTimeDomainPlots();
-        // Button to close popup.
-        if (ImGui::Button("Close")) {
-          ImGui::CloseCurrentPopup();
-        }
-        ImGui::EndPopup();
+      ImGui::BeginChild("Diagnostic Plots");
```

Comment:
If we change this from `ImGui::BeginChild()` to `ImGui::Begin()`, we can put these in a separate window to make things less busy. Seems to work fine, and the only thing extra we need to do is set the window size.

Reply:
Would we move this outside of the Feedforward Dropdown so that the window would pop up at the beginning of the program?

Reply:
Whichever you think makes sense. I would prefer that, yes.